woocommerce/woocommerce-android
# Feature/9435 notification survey screen

⚠️ Do NOT merge until `trunk` is the base branch for this PR.

Part of #9435


### Description

Displays a survey upon tapping on the 24h local notification scheduled after subscribing to free trial plan. 

### Testing instructions

1. Go to[ line 85 from LocalNotifications.kt](https://github.com/woocommerce/woocommerce-android/pull/9444/files#diff-74c116a16145d5ea7d01758d98557b5a20c465d60e70c9e45b59373b810b057bR97) and change HOURS for SECONDS. Maybe even reduce the 24 number to 1 or 2 seconds. 
3. Wait until the new store is ready for the notification to be shown. Check it displays the correct title and body. Like in the screen recording below.
4. Tap on it and check the survey is displayed. 
5. Click on any of the options. 
6. Before tapping send, click on the `Other reason` option. Type some text and notice how the button to send the feedback is enabled
7. Send the feedback and check the following event is tracked: 
`Tracked: free_trial_survey_sent, Properties: {"survey_option":"other_reasons","free_text":"test c","blog_id":221504240,"is_wpcom_store":true,"is_debug":true}`
8. The survey screen should be dismissed after sending the answer


### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/2663464/ec476a81-926d-4789-a743-5f5696204f62
